lib/svg.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:ui';
+import 'package:flutter_svg/flutter_svg.dart';
+
+import 'flame.dart';
+import 'position.dart';
+
+class Svg {
+  DrawableRoot svgRoot = null;
+  Size size;
+
+  Svg(String fileName) {
+    Flame.assets.readFile(fileName).then((svgString) async {
+      this.svgRoot = await svg.fromSvgString(svgString, svgString);
+    });
+  }
+
+  render(Canvas canvas, double width, double height) {
+    svgRoot.scaleCanvasToViewBox(canvas, Size(width, height));
+    svgRoot.draw(canvas, null);
+  }
```

Comment:
the Sprite class has a few more render options than this.
maybe that doesn't make sense in this case, but what about supporting them?
like, render taking in a Position size, render taking in x,y position, etc
I know you won't use those in the SvgComponent class, but for someone who wishes to use the Svg class directly, might be of some help

Reply:
If you disagree that that's necessary for this case, you can safely resolve this comment, I don't stand by this ;) this was more of a heads-up